asset/js/setup.js

```diff
@@ -19,6 +19,18 @@ function flattenModals() {
   });
 }
 
+/**
+ * Fixing the top navbar would break anchor navigation,
+ * by creating empty spans above the <h> tag we can prevent
+ * the headings from being covered by the navbar.
+ */
+function setupDummySpans() {
+  jQuery('<span class="anchor"></span>').insertBefore('h1, h2, h3, h4, h5, h6');
+  jQuery('span[class="anchor"]').each((index, element) => {
+    jQuery(element).attr('id', jQuery(element).next().attr('id'));
```

Comment:
This leads to both the dummy span and the actual heading having the same `id`.  I think we should try to avoid this.
<img width="305" alt="Screenshot 2020-01-17 at 8 42 47 AM" src="https://user-images.githubusercontent.com/17447681/72574805-59a7a480-3905-11ea-9a64-59b3297a6ce3.png">

Reply:
I removed the id for all headings (if dummy spans are added as their anchors). Is this OK?